helix-view/src/editor.rs

```diff
@@ -1753,12 +1756,16 @@ impl Editor {
                     if  !self.needs_redraw{
                         self.needs_redraw = true;
                         let timeout = Instant::now() + Duration::from_millis(33);
-                        if timeout < self.idle_timer.deadline(){
-                            self.idle_timer.as_mut().reset(timeout)
+                        if timeout < self.idle_timer.deadline() && timeout < self.redraw_timer.deadline(){
```

Comment:
Do we still need to check the idle_timer here or can we switch this to only look at the redraw_timer?